packages/test/test-end-to-end-tests/src/test/attachLifecycle.spec.ts

```diff
@@ -56,9 +56,9 @@ describeFullCompat("Validate Attach lifecycle", (getTestObjectProvider) => {
             const provider = getTestObjectProvider();
             const timeoutDurationMs = this.timeout() / 2;
             let containerUrl: IResolvedUrl | undefined;
-            const oldRegistry: [string | undefined, IChannelFactory][] =
+            const sharedStringRegistry: [string | undefined, IChannelFactory][] =
```

Comment:
this registry isn't specific to the sharedString, it is just a dds registry and only happens to have a shared string within it

Reply:
I accidentally named it it "oldRegistry" in a prior change. The "old" part makes no sense.